tests/course/test_for_review_course.py

```diff
@@ -23,3 +23,18 @@ def test_student_reviewer(self, persist_review):
     def test_get_number_of_review_of_course(self, persist_review, persist_review_number_two):
         result = Review.objects.get_number_of_review_of_course(persist_review.course)
         assert result == 2
+
+    def test_check_if_review_exist_by_student(self, persist_review, persist_second_course):
+        review = Review.objects.check_if_review_exist_by_student(persist_review.student, persist_review.course)
+        assert review is not None
+        review = Review.objects.check_if_review_exist_by_student(persist_review.student,
+                                                                 persist_second_course.course_id)
+        assert review is None
+
+    def test_check_if_review_made_by_student(self, persist_review):
```

Comment:
This should be a parameterized test with a use case where the review is *not* made by the student

Reply:
> This should be a parameterized test with a use case where the review is _not_ made by the student

This test was deleted